client/src/dependencies/DependencyManager.ts

```diff
@@ -105,7 +105,10 @@ export class DependencyManager {
 
                 } catch (error) {
                     outputAdapter.error(`Could not rebuild native dependencies ${error.message}`);
+                    outputAdapter.error('Please ensure that you have node and npm installed');
                     vscode.window.showErrorMessage(`Could not rebuild native dependencies ${error.message}`);
+                    vscode.window.showErrorMessage('Please ensure that you have node and npm installed');
```

Comment:
Does this look right when you run it?

Reply:
I just ended up combining the two error messages together to show that they're related.